bin/git-stats

```diff
@@ -18,35 +18,47 @@ Moment.suppressDeprecationWarnings = true;
 switch (process.argv[2]) {
     case "--record":
         var data = process.argv[3].replace(/^\"|\"$/g, "");
-        try {
-            data = JSON.parse(data);
-        } catch (e) {
-            throw e;
-        }
+        data = JSON.parse(data);
+
         GitStats.record(data, function (err) {
             if (err) { return Logger.log(err, "error"); }
         });
         break;
     case "--help":
+    case "-h":
         console.log(Fs.readFileSync(__dirname + "/docs/help", "utf-8"));
         break;
     case "-v":
         console.log(require("../package.json").version);
         break;
     default:
-        var options = {};
-
-        if (process.argv.length !== 2) {
-            if (process.argv[2]) {
-                options.start = Moment(process.argv[2]);
+        var options = {
+                theme: "DARK"
+            },
```

Comment:
Please use [this comma style](https://github.com/IonicaBizau/code-style#commas).

```js
var options = {
       ...
    }
  , parameter = null
  , i = 0
  ;
```